source/patterns/@aws-solutions-constructs/core/lib/glue-job-helper.ts

```diff
@@ -99,41 +107,55 @@ export function buildGlueJob(scope: Construct, props: BuildGlueJobProps): [glue.
         'it is recommended to use "WorkerType" or  "NumberOfWorkers"');
       }
 
-      return deployGlueJob(scope, props.glueJobProps, props.database!, props.table!, props.outputDataStore!, props.etlCodeAsset);
+      const deployGlueJobResponse =
+        deployGlueJob(scope, props.glueJobProps, props.database!, props.table!, props.outputDataStore!, props.etlCodeAsset);
+      return {
+        job: deployGlueJobResponse.job,
+        role: deployGlueJobResponse.role,
+        bucket: deployGlueJobResponse.bucket,
+        loggingBucket: deployGlueJobResponse.loggingBucket };
```

Comment:
Can this be simplified to just: 

```
return deployGlueJob(scope, props.glueJobProps, props.database!, props.table!, props.outputDataStore!, props.etlCodeAsset);
```

The fields look the same at a glance, but maybe they aren't the same types or something, let me know.

Reply:
Yes, that would work and is what happened in the past. But they are different types that coincidentally have the same attributes. To return one type in place of the other because we can currently get away with it would be brittle and couple the two routines.